migrations/versions/f448930e4058_.py

```diff
@@ -0,0 +1,33 @@
+"""create table identity_registration
+
+Revision ID: f448930e4058
+Revises: 6a151731efac
+Create Date: 2019-03-04 15:50:52.395995
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = 'f448930e4058'
+down_revision = '6a151731efac'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'identity_registration',
+        sa.Column('identity', sa.String(length=42), nullable=False),
+        sa.Column('created_at', sa.DateTime(), nullable=True),
+        sa.Column('updated_at', sa.DateTime(), nullable=True),
+        sa.Column('payout_eth_address', sa.String(length=42), nullable=True),
+        sa.Column('signed_body', sa.String(length=1023), nullable=True),
+        sa.Column('signature', sa.String(length=255), nullable=True),
```

Comment:
I don't really see the need for signed_body and the signature fields.

Reply:
removed